docs/src/_includes/layouts/doc.html

```diff
@@ -34,6 +34,17 @@ <h2 id="further-reading">Further Reading</h2>
 
         {% set all_content = [all_content, further_reading_content] | join %}
     {% endif %}
+        
+    {% if rule_type %}
+        {% set resources_content %}
+            <h2 id="resources">Resources</h2>
+            <ul>
+                <li><a href="https://github.com/eslint/eslint/blob/main/lib/rules/{{ title }}.js">Rule source</a></li>
```

Comment:
On the current website, we're using `HEAD` instead of `main`. Just to check - did you intentionally switch to `main`?

Reply:
Yes. We used HEAD to aid with the transition from master to main. I don’t think it’s necessary anymore.